charts/mattermost-enterprise-edition/templates/deployment-mattermost-hpa.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.mattermostApp.autoscaling.enabled }}
-apiVersion: autoscaling/v2beta1
+apiVersion: autoscaling/v2
```

Comment:
We need a conditional check here. For example:
```
{{- if semverCompare ">= 1.25-0" .Capabilities.KubeVersion.Version }}
apiVersion: autoscaling/v2
{{ else }}
apiVersion: autoscaling/v2beta1
```

With this ways it's backward compatible.

Reply:
@spirosoik updated it

Reply:
awesome. Thanks @jonathanwiemers